pkg/network/types.go

```diff
@@ -40,6 +50,25 @@ func GetNetworkInfoForEC2Instances(ec2svc ec2iface.EC2API, nodeInstanceIds []str
 		}
 	}
 
+	for asgName := range asgs {
+		asgOutput, err := autoscalingSvc.DescribeAutoScalingGroups(&autoscaling.DescribeAutoScalingGroupsInput{
+			AutoScalingGroupNames: aws.StringSlice([]string{asgName}),
```

Comment:
If the function takes in a slice as input, why are we making a individual calls

Reply:
I think we could. Might be paging is one thing to consider though it is very rare for this use case.
> The maximum number of items to return with this call. The default value is 50 and the maximum value is 100.
from https://docs.aws.amazon.com/autoscaling/ec2/APIReference/API_DescribeAutoScalingGroups.html